cns/common/service.go

```diff
@@ -35,20 +36,22 @@ type ServiceConfig struct {
 	Listener *acn.Listener
 	ErrChan  chan error
 	Store    store.KeyValueStore
+	Managed  bool
```

Comment:
Instead of defining a bool, can we define it as ChannelType? (with values as Direct, Managed) Later that can be extended to CRD as well

Reply:
I have no context on this, why channel?

Reply:
its not go channel..she s asking you to define enum of strings..so that it can take multiple values instead of bool

Reply:
in dnc there's already direct and privatelink channel modes (for cns), are they related to this at all?

e.g. managed here could mean privatelink on dnc side but it may not be guaranteed